<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>WsDirectory</class>
 <widget class="QWidget" name="WsDirectory" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WengoPhone - Search for Wengo contacts</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QComboBox" name="criteriaComboBox" >
     <item>
      <property name="text" >
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Nickname</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>First Name</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Last Name</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>City</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Country</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="searchPushButton" >
     <property name="text" >
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLineEdit" name="searchLineEdit" />
   </item>
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="searchLabel" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>10</red>
         <green>36</green>
         <blue>106</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>233</red>
         <green>231</green>
         <blue>227</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>10</red>
         <green>36</green>
         <blue>106</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <family>MS Shell Dlg</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>&lt;span style="font-size:18pt;">Search for Wengo contacts&lt;/span>&lt;br />&lt;br />&lt;span>Enter a name, nickname, city, or email and click the search button&lt;/span></string>
     </property>
     <property name="indent" >
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QListWidget" name="listWidget" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>236</red>
         <green>238</green>
         <blue>249</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>236</red>
         <green>238</green>
         <blue>249</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>233</red>
         <green>231</green>
         <blue>227</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>213</red>
         <green>213</green>
         <blue>213</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>236</red>
         <green>238</green>
         <blue>249</blue>
        </color>
       </disabled>
      </palette>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="midLineWidth" >
      <number>0</number>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>When entering a combined search, please separate words with a blank.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>searchLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>165</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel" >
     <x>371</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
